demo-html/index.html

```diff
@@ -13,55 +13,246 @@
+                    <span style="color: rgb(236, 233, 20);">
+                        <i class="fas fa-shopping-cart fa-10x"></i>
+                    </span>
+                    <h3><br>Comprá</h3>
+                    <p class="textService">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem eaque eius voluptas!
+                        A, qui quia! Quaerat itaque commodi aspernatur labore?
+                    </p>
+                </div>
+                <div class="col-sm">
+                    <span style="color: rgb(236, 233, 20);">
+                        <i class="fas fa-laptop fa-10x"></i>
+                    </span>
+                    <h3><br>Recibí</h3>
+                    <p class="textService">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem eaque eius voluptas!
+                        A, qui quia! Quaerat itaque commodi aspernatur labore?
+                    </p>
+                </div>
+                <div class="col-sm">
+                    <span style="color: rgb(236, 233, 20);">
+                        <i class="fas fa-tshirt fa-10x"></i>
+                    </span>
+                    <h3><br>Organizate</h3>
+                    <p class="textService">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem eaque eius voluptas!
+                        A, qui quia! Quaerat itaque commodi aspernatur labore?
+                    </p>
+                </div>
+            </div>
         </div>
+        <br><br><br><br>
     </section>
 
     <!-- Portfolio -->
     <section class="page-section" id="portfolio">
-        <div class="container">
+        <div class="container-portfolio">
+            <h2><br><br><br><strong>PORTFOLIO</strong></h2>
+            <p>Lorem ipsum dolor sit amet consectetur<br><br></p>
+            <div class="row">
+                <div class="col-sm">
+                    <img src="img/portfolio/01-thumbnail.jpg">
+                    <h3><br>Prendas</h3>
+                    <p class="textPortfolio">Ejemplo<br><br></p>
+                    <img src="img/portfolio/04-thumbnail.jpg">
+                    <h3><br>Mas cosas</h3>
+                    <p class="textPortfolio">Marca</p>
+                </div>
+                <div class="col-sm">
+                    <img src="img/portfolio/02-thumbnail.jpg">
+                    <h3><br>Explora</h3>
+                    <p class="textPortfolio">Cosas<br><br> </p>
+                    <img src="img/portfolio/05-thumbnail.jpg">
+                    <h3><br>UCEMA</h3>
+                    <p class="textPortfolio">Diseño </p>
+                </div>
+                <div class="col-sm">
+                    <img src="img/portfolio/03-thumbnail.jpg">
+                    <h3><br>Proyectos</h3>
+                    <p class="textPortfolio">Identidad<br><br> </p>
+                    <img src="img/portfolio/06-thumbnail.jpg">
```

Comment:
Siempre que pongan una imagen, pongan también un atributo `alt` con una descripción breve de qué contiene la imagen. De esa forma permiten que personas con visibilidad reducida puedan leer más fácilmente su sitio usando un lector de pantalla: 

https://youtu.be/7Rs3YpsnfoI?t=33